src/utils/components/AddBootableVolumeModal/AddBootableVolumeModal.tsx

```diff
@@ -52,16 +50,17 @@ const AddBootableVolumeModal: FC<AddBootableVolumeModalProps> = ({
 }) => {
   const { t } = useKubevirtTranslation();
   const [activeNamespace] = useActiveNamespace();
-  const selectedNamespace =
```

Comment:
why did. removed it? if i remember correctly we added it for a reason, but I cant remember the reason :)

Reply:
I didn't fully remove it. I just moved it to the BootableVolume object. This allows the namespace to be pulled from it since it's already passed around.